### PR TITLE
fix(components): [el-select] fix composition bug #3532

### LINKED ISSUE #3532: [Bug Report] El Select options changed when composition
### Element Plus version
1.1.0-beta.12

### OS/Browsers version
Mac OS / Chrome 93.0.4577.63

### Vue version
3.2.12

### Reproduction Link
https://codepen.io/bchen1222/pen/NWdoYvG?editors=1000

### Steps to reproduce
1. 新增一個 el-select
2. 將 filterable 設為 true
3. 新增有中文 label 的選單，
options: [
{
value: 'cake',
label: '蛋糕'
}
...
]

### What is Expected?
可以正確 filter 符合輸入內容的選項

### What is actually happening?
使用中文輸入法上下鍵選字時，選單的 active 選項也會跟著上下滑動，導致無法選到正確的中文字

### PR DESCRIPTION
**Pull request by bchen1029**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

fix(components): [el-select] fix composition bug #3532

### CODE REVIEW

File: packages/components/input/src/index.vue [hunk 1]
```diff
@@ -434,17 +437,20 @@ export default defineComponent({
       inputOrTextarea.value.select()
     }
 
-    const handleCompositionStart = () => {
+    const handleCompositionStart = (event) => {
```
Comment (JeremyWuuuuu) [inline packages/components/input/src/index.vue:440]: **Nitpick:** would you mind annotating the `event`'s type?

**Review by jw-foss — APPROVED** [review]
I left a non-blocking comment for you to consider, happy to approve as-is

File: packages/components/input/src/index.vue (hunk 1 above)
Comment (bchen1029) [reply packages/components/input/src/index.vue]: done.

**Comment by charlie0228** [comment]
@JeremyWuuuuu He also opens a pull request at https://github.com/ElemeFE/element/pull/21336 .
Could someone help to approve the PR?

**Pull request merged**